contracts/btc-staking/src/state/delegations.rs

```diff
@@ -0,0 +1,102 @@
+use crate::state::points_alignment::PointsAlignment;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{CanonicalAddr, Order, StdResult, Storage, Uint128};
+use cw_storage_plus::{Index, IndexList, IndexedMap, KeyDeserialize, MultiIndex};
+
+/// Single delegation related information - entry per `(staking hash, finality provider public key)`
+/// pair, including distribution alignment
+#[cw_serde]
+pub struct Delegation {
+    /// The delegator's canonical address
+    pub staker_addr: CanonicalAddr,
+    /// How many satoshis the user stakes in this delegation
+    pub stake: u64,
+    /// How many points should be added / subtracted from points calculated per delegation due to
+    /// delegation changes.
+    pub points_alignment: PointsAlignment,
```

Comment:
Not sure if I fully under this definition -- 

1. What is the definiton of "point" in this context?
2. Why would "the amount of points added/substracted" be a property of a delegation that is not affected by external factors?

Reply:
This has been refactored out, as it's not needed in our current impl.